src/Storage.js

```diff
@@ -0,0 +1,69 @@
+import redis from 'redis'
+
+class Storage {
```

Comment:
yeah could improve this, and some additional docs on implementing your own storage interface should definitely be provided